sub/source/docs/introduction.rst

```diff
@@ -4,15 +4,36 @@
 Introduction to Sub
 ===================
 
-**UNDER CONSTRUCTION**
+Sub is an advanced open-source autopilot system for submersible ROVs (Remote Operated Vehicle) that supports multiple vehicle configurations. It offers a variety of operating modes from fully manual to fully autonomous.
 
+.. image:: ../images/sub-system.jpg
+    :target: ../_images/sub-system.jpg
 
 
-Vehicle Configuration Types
-===========================
-
+Vehicle Configuration Options
+=============================
 
+* Sub supports both brushed and brushless, uni-directional or bi-directional propulsion motors/thrusters
+* 3/4/5 or 6 thruster configurations controlling up to all 6 degrees of vehicular motion and axial rotation
+* Pilot control and telemetry via cable tether to a Ground Control Station (GCS) using joystick, button, and/or keyboard commands, or
+* Pilot control and telemetry via radio control to tethered surface buoy
+* Position and Depth Control using optional depth sensor, GPS while surfaced, Sonars, or Shore based acoustic beacon locators
+* Leak, Temperature, and internal Pressure sensor options for vehicle safety
+* Ethernet vehicle peripheral connectivity option for sophisticated sensors like Side Scan sonars
+* 100's of ArduPilot compatible autopilots to choose from
+* Optional Linux based companion computer interface to autopilot for image processing, beacon data processing, sonar displays in the GCS, custom applications, etc.
+* Analog and Digital Video camera and OSD (On Screen Display) support
+* Optional Buoyancy Control instead of vertical thrusters/neutral buoyancy
```

Comment:
I dont think we have any explicit buoyancy control... yet.

Reply:
could be rigged with motor/syringe combo off the vertical motor output channel ...nothing in code to prevent that being used for vertical instead of a thruster